roles/matrix-synapse/defaults/main.yml

```diff
@@ -426,3 +426,47 @@ matrix_synapse_configuration_extension: "{{ matrix_synapse_configuration_extensi
 # Holds the final Synapse configuration (a combination of the default and its extension).
 # You most likely don't need to touch this variable. Instead, see `matrix_synapse_configuration_yaml`.
 matrix_synapse_configuration: "{{ matrix_synapse_configuration_yaml|from_yaml|combine(matrix_synapse_configuration_extension, recursive=True) }}"
+
+
+# oidc / sso
+
+# Set to false if you need only SSO auth
+matrix_synapse_password_config_enabled: true
```

Comment:
Maybe this variable should be moved closer to `matrix_synapse_password_config_localdb_enabled`, because they belong together. It may be nice to have a comment about it there, because it's a little confusing which one does what.